src/java.base/share/classes/jdk/internal/foreign/CABI.java

```diff
@@ -36,10 +36,10 @@
 import static sun.security.action.GetPropertyAction.privilegedGetProperty;
 
 public enum CABI {
-    SysV,
-    Win64,
-    LinuxAArch64,
-    MacOsAArch64,
+    SYS_V,
+    WIN_64,
+    LINUX_AARCH_64,
+    MAC_OS_AARCH_64,
     SysVPPC64le,
     SysVS390x,
```

Comment:
These names should follow the convention of other literals defined here (i.e. `SYS_V_PPC64LE` and `SYS_V_S390X`).

Reply:
The intention is to minimize the FFI related changes to avoid modify the code everywhere so as to ensure it passes the compilation in JDK20. All FFI related code will be double-checked & updated (including the naming here)  after this temp changes are merged.

Reply:
Please (create and) share the URL of an issue that will track that work.

Reply:
We already have the issue for all FFI related code in JEP434/JDK20 via https://github.com/eclipse-openj9/openj9/issues/16329 to track the work. I already updated the PR description for this.